Languages/Japanese/DefInjected/ResearchProjectDef/ResearchProjects.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
+  <!-- MedPod Research Projects -->
+
+  <AtomicCellularRegeneration.label>分子細胞再生</AtomicCellularRegeneration.label>
+  <AtomicCellularRegeneration.description>生きている細胞を原子レベルで迅速に修復することにより、様々な病気や怪我を自動的に治療することができる高度な病院ベッドである治療ポッドの建設を可能にします。</AtomicCellularRegeneration.description>
+
```

Comment:
Could you also translate the pop-up letter title and description?

```
<!-- EN: About: Building MedPods -->
<AtomicCellularRegeneration.discoveredLetterTitle>???</AtomicCellularRegeneration.discoveredLetterTitle>
	
	<!-- EN: You now have the technology to build MedPod beds - but each MedPod requires a special Isolinear Processor component. You can collect Isolinear Processors several ways:\n\n  -Orbital or caravan exotic goods traders sell Isolinear Processors. Invite or wait for them.\n  -Some quests are rewarded with Isolinear Processors.\n    -Disassemble an AI persona core to extract Isolinear Processors (along with various other components). -->
<AtomicCellularRegeneration.discoveredLetterText>???</AtomicCellularRegeneration.discoveredLetterText>
```

Reply:
I missed the research letter. I'll add it immediately.